api/api.py

```diff
@@ -206,6 +207,10 @@ class OutputExpensesRevenueYearFilterSerializer(serializers.Serializer):
     expense_group = serializers.CharField()
 
 
+def requests(country):
```

Comment:
Remove unused functions. Also 'requests' is a reserved word. Never use it as a function

Reply:
done